semconvgen/generator.go

```diff
@@ -162,13 +164,20 @@ func render(cfg config) error {
 
 	args := []string{
 		"run", "--rm",
-		"-v", fmt.Sprintf("%s:/data", tmpDir),
+		"-v", fmt.Sprintf("%s:/data:z", tmpDir),
```

Comment:
This seems unrelated.  What does this change will do?  A comment might be helpful.

Reply:
Enforced selinux policies can cause issues when files get shared with a containerized process. Since file permissions are reflected by this labels[0]. Docker/Podman have the ability to relabel[1] shared files. 

Should I remove this from this PR and change it in another one or add a comment on top? What would you recommend?

0: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security-enhanced_linux/sect-security-enhanced_linux-working_with_selinux-selinux_contexts_labeling_files
1: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label

Reply:
Let's address that in a separate PR.

Reply:
Its removed from this one. :)